bot/skills/__init__.py

```diff
@@ -36,16 +35,27 @@ def _add_version(upd: Updater, version_handlers_group: int):
                                   filters=admin_filter), version_handlers_group)
 
 
+def _get_version_from_toml() -> str:
```

Comment:
Semantically, the name of this function tells us it could get the version from any toml. 

So, it should call either  `_get_version_from_pyproject()` or `_parse_toml(path: str) -> dict` and extract `version` in the `_version` handler from the dict.

Reply:
Changed.